### ISSUE TITLE
Not displaying field values in manager when MODX installed in sub-directory

### ISSUE THREAD

**Issue by digitalpenguin** [goldsky/Lingua#42]
If MODX is installed in web root everything works fine. 

However, if MODX is installed in a sub directory, when switching from the default language to another on a resource in the manager all the fields are blank. 

I seem to be able to save and everything works on the web context if the fix is applied from the previous issue, but when you come back to the secondary language in the manager all the fields are blank. It seems to have trouble loading the values even though they are apparently saved. 

I haven't tested TVs yet.

**Comment by digitalpenguin** [comment]
Not just sub directory installations.
I'm having the same problem in my latest modx root installation. :(

**Comment by digitalpenguin** [comment]
I believe this issue is to do with languages that share the same language code.

For example I had 5 newly created resources.
I then installed Lingua.
There is the Chinese language with an id of 6. language code is 'zh' and the LCID string is 'zh-cn'

I created a new language for Hong Kong style Chinese. language code 'zh' LCID string is 'zh-hk'
I made this active in addition to English. Was given an id of 9.

The language switcher on the resource manager page shows the correct languages but when I looked in the database table, it showed the data I entered was going into the language with id 6.
So it must be getting confused somewhere...
